jjc

```diff
@@ -74,11 +74,11 @@ BEGIN {
 
 	# Try to create the "channels" directory while also creating all
 	# parent directories as needed.
-	local["retval"]=system("mkdir -p -- \"$IRC_DIR/$IRC_HOST/channels\" >&2")
+	local["retval"]=system("exec mkdir -p -- \"$IRC_DIR/$IRC_HOST/channels\" >&2")
```

Comment:
Exceeds 74 columns. The additional `exec` also makes it too long even when putting the `system()` call on the next line. We could use a variable for the command instead.